compatibility/bazel_tools/testing.bzl

```diff
@@ -98,6 +98,31 @@ excluded_test_tool_tests = [
             },
         ],
     },
+    {
+        "end": "1.3.0-snapshot.20200617.4484.0.7e0a6848",
+        "platform_ranges": [
+            {
+                "start": "0.0.0",  #fix to first snapshot after now
+                "exclusions": [
+                    # See https://github.com/digital-asset/daml/pull/7400
```

Comment:
I couldn't find anything on this PR explaining what is going on here and below. Could you please leave a top-level comment with an explanation?

Reply:
PR description extended